core/controllers/admin.py

```diff
@@ -100,9 +100,8 @@ def get(self):
         recent_job_data = jobs.get_data_for_recent_jobs()
         unfinished_job_data = jobs.get_data_for_unfinished_jobs()
         for job in unfinished_job_data:
-            job['can_be_canceled'] = job['is_cancelable'] and any([
-                klass.__name__ == job['job_type']
-                for klass in jobs_registry.ONE_OFF_JOB_MANAGERS])
+            job['can_be_canceled'] = job['is_cancelable'] and any( klass.__name__ == job['job_type']
```

Comment:
Ah -- I like the idea, but these and the other related changes need to go in a separate PR entirely. They aren't related to this one.